### ISSUE TITLE
The gesture mappings for the Hims Braille Edge follow the global setting and not the model-specific map

### ISSUE THREAD

**Issue by Piciok** [nvaccess/nvda#8096]
The user guide as well as the NVDA settings both mention two alternative gesture mappings for the hims displays: a global one and one specific to the model of the display.

I'm using a Braille Edge and currently NVDA behaves as if the global setting was the one to be used when in fact the Braille Edge should be. This leads to conflicts, E.G. the gesture for the ctrl key assigned to Braille Edge displays is the third function key, F3; however, when I press it, NVDA records and executes the tab key command.

I've checked it with a fresh, portable copy of 2018.1 with no custom settings.

**Comment by leonardder** [comment]
I'm going to investigate this next week. I consider this a personal p2, given it is an annoying bug and probably not very difficult to fix. I will self assign this.

**Comment by leonardder** [comment]
Could you please report whether [this](https://ci.appveyor.com/api/buildjobs/cyv562936hfrhj91/artifacts/output%2Fnvda_snapshot_try-i8096-14955%2C8d90b265.exe) try build fixes your issue?

**Comment by Piciok** [comment]
That would be it! Works like a charm! Thanks a million!
